RELEASE.md

```diff
@@ -1,3 +1,11 @@
+# Release 1.13.1
+
+## Bug fixes
+- Fix launch error on Windows (#1900)
```

Comment:
If these fixed a bug that was actually filed, I list that first, and then put the PR fixing it as `(PR #NNNN)` after that.  If no bug I list the PR first as being the documentation for the bug.  See 1.13.0 for the usual style.

So in this case that would be #1895 fixed by #1900, and #1794 fixed by #1898.  For the profile dashboard I guess we could use #1913 but probably makes sense to just reference #1914.

I think #1945 can be broken out as a separate fix with no attached bug, since it fixes an issue that's not inherently related to Colab and was observable before (by compressing the viewport enough).

Finally, "launch error" is pretty vague - perhaps "Fix strftime-related launch error on Windows"?

Reply:
> If these fixed a bug that was actually filed, I list that first, and
> then put the PR fixing it […] If no bug I list the PR first as being
> the documentation for the bug

Ah, so the first token is _either_ a PR or an issue depending on the
line item—okay, I understand now. Done. (Rest done as well.)

Reply:
Yes... I won't argue it's the most straightforward precedent ever, but that is what I've been doing.  I'm open to alternative suggestions though as long as we commit to being consistent with some convention.  Mostly the thing I'd like is to include the bug reference itself in the text when fixing a numbered bug so that it's clear from the relnotes without having to visit the PR first.

An alternative convention that's maybe clearer would just be to insert the bug number in the prose text somewhere, like:

```
- Fix #ISSUE strftime-related startup error on Windows (#PR)
- Disable profile dashboard in Colab where it doesn't work yet (#PR)
```

Reply:
Yep, no worries; consistency is good, and this is fine with me.